### DISCUSSION TITLE
paopaodns能否指定某个域名返回AAAA记录？

### DISCUSSION THREAD

**Discussion by sk163** [kkkgo/PaoPaoDNS#177] [想法]
paopaodns能否指定某个域名返回AAAA记录？不论这个域名是国内或国外，只要有AAAA就返回。目前设置的是yes_only6或only6，有的国内的域名即使有AAAA也不想返回，如果没有A的再返回，这个用only6能解决，但是还有一些国外的双栈vps想返回AAAA，能否通过某种配置，指定某个域名是否返回AAAA？

**Comment by kkkgo** [comment]
用custom_mod定义可以最高优先级，转发到udp://127.0.0.1:5302

**Reply by sk163** [reply]
好嘞，多谢～
